### ISSUE TITLE
energy transmitter/reciever not functioning

### ISSUE THREAD

**Issue by mrcoffee1026** [J3fftw1/SlimeXpansion#14]
## Description (Required)
I don't know what range these are supposed to have, but they would have at least the 6 block range that electrical connections have.  Having them this close together, they do not transmit nor receive any energy (the transmitter does - however - use up energy).  Pretty sure any placed transmitter should automatically connect with any receiver that has been placed in the same world, never had a chance to verify that for certain though.

## Steps to reproduce the Issue (Required)
Place near together and check receiver with a multimeter, always shows 0 J / 12K J while transmitter shows 12K J / 12K J.  
https://imgur.com/a/5vupnEU

## Expected behavior (Required)
These two devices should connect to each other and send power remotely for some *unknown to me* distance.

## Server Log / Error Report
No errors appearing for this, they just aren't functioning as items.

## Description (Required)
I don't know what range these are supposed to have, but they would have at least the 6 block range that electrical connections have.  Having them this close together, they do not transmit nor receive any energy (the transmitter does - however - use up energy).  Pretty sure any placed transmitter should automatically connect with any receiver that has been placed in the same world, never had a chance to verify that for certain though.

## Steps to reproduce the Issue (Required)
Place near together and check receiver with a multimeter, always shows 0 J / 12K J while transmitter shows 12K J / 12K J.  
https://imgur.com/a/5vupnEU

## Expected behavior (Required)
These two devices should connect to each other and send power remotely for some *unknown to me* distance.

## Server Log / Error Report
No errors appearing for this, they just aren't functioning as items.

## Environment (Required)
[13:19:03 INFO]: Paper v1_15_R1 (161)
[13:19:03 INFO]:
[13:19:03 INFO]: CS-CoreLib vDEV - 83 (git 8081bb4f)
[13:19:03 INFO]: Slimefun vDEV - 477 (git 156683f3)
[13:19:03 INFO]:
[13:19:03 INFO]: Installed Addons (18)
[13:19:03 INFO]:  DyedBackpacks vDEV - 5 (git 129670c7)
[13:19:03 INFO]:  ExtraTools vDEV - 4 (git 02799dac)
[13:19:03 INFO]:  SoundMuffler vDEV - 10 (git 545192ca)
[13:19:03 INFO]:  SlimefunOreChunks vDEV - 8 (git 41e31ffc)
[13:19:03 INFO]:  SlimeXpansion vDEV - 14 (git 0fa0ac7b)
[13:19:03 INFO]:  EmeraldEnchants vDEV - 5 (git 6adbbb87)
[13:19:03 INFO]:  HotbarPets vDEV - 26 (git 7489ae00)
[13:19:03 INFO]:  ExtraGear vDEV - 7 (git 9e953653)
[13:19:03 INFO]:  ExoticGarden vDEV - 38 (git a6eabd5a)
[13:19:03 INFO]:  Barrels vDEV - 11 (git abecba9c)
[13:19:03 INFO]:  ElectricSpawners vDEV - 10 (git 6c891f7f)
[13:19:03 INFO]:  ExtraHeads vDEV - 13 (git 88072fdf)
[13:19:03 INFO]:  SoulJars vDEV - 12 (git 6b475952)
[13:19:03 INFO]:  SlimyTreeTaps vDEV - 10 (git 66d1dd72)
[13:19:03 INFO]:  MobCapturer vDEV - 7 (git b61d3fbf)
[13:19:03 INFO]:  PrivateStorage vDEV - 11 (git b064863b)
[13:19:03 INFO]:  ColoredEnderChests vDEV - 11 (git 7374efd1)
[13:19:03 INFO]:  ChestTerminal vDEV - 30 (git 04393688)

You closed this as ticket #4 but didn't check the results this issue which still persists through much newer builds of this addon and of slimefun.

**Comment by J3fftw1** [comment]
i have no clue how this thing works i fixed something to the energy reciever but i have no clue how this thing works so i hope john can fix it

**Comment by mrcoffee1026** [comment]
yeah i'm barely aware of the limitations it should have or anything myself but i am absolutely certain the transmitter should not open to what looks like a chest inventory... and having one within range of a receiver should charge provide the receiver with power as that's the whole point but I'm not sure what the range is supposed to be.
